src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/serializer.html.twig

```diff
@@ -0,0 +1,184 @@
+            <span class="label">Deserializations</span>
+        </div>
+        <div class="metric">
+            <span class="value">{{ collector.normalizations | length }}</span>
+            <span class="label">Normalizations</span>
+        </div>
+        <div class="metric">
+            <span class="value">{{ collector.denormalizations | length }}</span>
+            <span class="label">Denormalizations</span>
+        </div>
+    </div>
+
+    <div class="sf-tabs">
+        <div class="tab {{ collector.serializations|length == 0 ? 'disabled' }}">
+            <h3 class="tab-title">Serializations <span class="badge">{{ collector.serializations | length }}</span></h3>
+
+            <div class="tab-content">
+                <h4>Serializations</h4>
+                <table>
+                    <thead>
+                    <tr>
+                        <th>#</th>
+                        <th>Format</th>
+                        <th>From</th>
+                        <th>To</th>
+                        <th>Context</th>
+                    </tr>
+                    </thead>
+                    <tbody>
+                    {% for serialization in collector.serializations %}
+                        <tr>
+                            <td class="font-normal text-small text-muted nowrap">{{ loop.index }}</td>
+                            <td class="nowrap">{{ serialization.format }}</td>
+                            <td>{{ profiler_dump(serialization.data, maxDepth=2) }}</td>
+                            <td>{{ serialization.result }}</td>
+                            <td>{{ profiler_dump(serialization.context, maxDepth=2) }}</td>
+                        </tr>
+                    {% endfor %}
+                    </tbody>
+                </table>
+            </div>
+        </div>
+        <div class="tab {{ collector.deserializations|length == 0 ? 'disabled' }}">
+            <h3 class="tab-title">Deserializations <span class="badge">{{ collector.deserializations | length }}</span>
+            </h3>
+
+            <div class="tab-content">
+                <h4>Deserializations</h4>
+                <table>
+                    <thead>
+                    <tr>
+                        <th>#</th>
+                        <th>Format</th>
+                        <th>Type</th>
+                        <th>From</th>
+                        <th>To</th>
+                        <th>Context</th>
+                    </tr>
+                    </thead>
+                    <tbody>
+                    {% for deserialization in collector.deserializations %}
+                        <tr>
+                            <td class="font-normal text-small text-muted nowrap">{{ loop.index }}</td>
+                            <td class="nowrap">{{ deserialization.format }}</td>
+                            <td class="nowrap">{{ deserialization.type }}</td>
+                            <td>{{ deserialization.data }}</td>
```

Comment:
same `profiler_dump`

Reply:
Opposite of serialization, the input of a deserialization should always be string, shouldn't it? Otherwise profiler_dump makes sense here, too.